src/test/operators/get_table_test.cpp

```diff
@@ -52,4 +56,55 @@ TEST_F(OperatorsGetTableTest, ExcludedChunks) {
   EXPECT_EQ(table->get_value<int>(ColumnID(0), 1u), original_table->get_value<int>(ColumnID(0), 3u));
 }
 
+TEST_F(OperatorsGetTableTest, ExcludeCleanedUpChunk) {
+  auto gt = std::make_shared<opossum::GetTable>("tableWithValues");
+  auto context = std::make_shared<TransactionContext>(1u, 3u);
+
+  auto original_table = StorageManager::get().get_table("tableWithValues");
+  auto chunk = original_table->get_chunk(ChunkID{0});
+
+  chunk->set_cleanup_commit_id(CommitID{2u});
+
+  gt->set_transaction_context(context);
+  gt->execute();
+
+  auto table = gt->get_output();
+  EXPECT_EQ(original_table->chunk_count(), 4);
+  EXPECT_EQ(table->chunk_count(), 3);
+}
+
```

Comment:
(Random line)

It seems like you're missing a test that excludes chunks both via `GetTable::set_excluded_chunk_ids()` and the pruning of deleted chunks.